storage/tianmu/core/engine.cpp

```diff
@@ -749,7 +750,8 @@ AttributeTypeInfo Engine::GetAttrTypeInfo(const Field &field) {
       }
       [[fallthrough]];
     default:
-      throw common::UnsupportedDataTypeException("Unsupported data type.");
+      throw common::UnsupportedDataTypeException(
+              std::string("Unsupported data type[") + common::get_enum_field_types_name(field.type()) + std::string("]"));
```

Comment:
Would you like to help us to improve this part code,, `Engine::CreateTable`.
The unspported Data type in everu corner, I think it's very very ugly impl, and should do this check within one place. Hence, this will make the logic more concise, and not be error-prone.

Reply:
ok，Take some time to familiarize with the code. The work can be a new issue and pull request